packages/terra-image/src/Image.jsx

```diff
@@ -70,7 +70,6 @@ const propTypes = {
 const defaultProps = {
   variant: 'default',
   isFluid: false,
-  alt: ' ',
```

Comment:
We should add a breaking change entry in the change log since `alt` will now truly be required.

Reply:
If this is a breaking change and a major version bump, do we want to merge this directly to main?

Reply:
[updated](https://github.com/cerner/terra-core/pull/3585/commits/f2c470f2fdc07522a43842aa21a82d87223662b1#diff-9128bd73892ba1ca56d7e9bc002df71fda83a19e40cb9f97f115b156ae39a318)